src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java

```diff
@@ -148,6 +153,18 @@ public void addContent(String content) {
     PageObjectLogging.log("addContent", "content " + content + " added to the article", true);
   }
 
+  public boolean checkPortableInfoboxVisible() {
+    try {
+      wait.forElementVisible(iframe);
+      verifyComponent(portableInfoboxTransclusion);
+      return true;
+    } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)